src/main/java/org/commcare/util/GetJsonProperty.java

```diff
@@ -0,0 +1,24 @@
+package org.commcare.util;
```

Comment:
This function needs to be available as an xpath expression, not only a java function, so that app builders have access to it. That means moving this into `src.main.java.org.javarosa.xpath.expr` and adding some boilerplate-type code to handle arguments. Check out the other classes in that package for examples, I think `XPathDistanceFunc` is the one most recently added.

Reply:
Ok I'll look at that example, thanks.